src/scenes/Wallet/components/Balances/index.jsx

```diff
@@ -70,6 +78,11 @@ class MyBalances extends React.PureComponent {
       <Paper className={classes.myBalancePaper}>
         <Grid container spacing={0} className={classes.myBalanceGridContainer}>
           <Typography variant="title" className={classes.myBalanceTitle}>
+            <div className={classes.importantNoteContainer}>
```

Comment:
move this `<div>` to `src/scenes/Wallet/index.jsx`. this does not belong in the MyBalances component.

Reply:
also wrap it in a `<Paper>` component when moving it. and move the marginBottom. so it looks like a separate piece of paper on the page.